docs/source/en/installation.md

```diff
@@ -48,6 +48,7 @@ pip install diffusers["torch"] transformers
 pip install diffusers["flax"] transformers
 ```
 </jax>
+Note - Pytorch doesn't support Python 3.12 on Windows. Please use Python 3.11 instead.
 </frameworkcontent>
```

Comment:
```suggestion
</jax>

Note - Pytorch doesn't support Python 3.12 on Windows. Please use Python 3.11 instead.

</frameworkcontent>
```

Reply:
Also can we move this statement into the PyTorch "frameworkcontent" section?

Reply:
Hi @patrickvonplaten, I have made another PR #6438. Please refer to that. I will close this one. Thanks